index_matroska.md

```diff
@@ -84,10 +84,18 @@ Attacks on a `Matroska Reader` could include:
 
 To be determined.
 
-# Notations and Conventions
+# Notation and Conventions
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
+This document defines specific terms in order to define the format and application of `Matroska`.  Specific terms are defined below:
+
+`Matroska`: a multimedia container format based on EBML (Extensible Binary Meta Language)
+
+`Matroska Player`: A `Matroska Player` is a data parser that interprets the semantics of a Matroska document and creates a way for programs to play `Matroska` documents as audiovisual files.
+
+`Matroska Reader`: A `Matroska Reader` is a data parser that interprets the semantics of a Matroska document and creates a way for programs to use `Matroska`.
```

Comment:
> to use `Matroska`

IMO it is too generic, I don't understand the idea behind having 2 names, what is the purpose of the `Matroska Reader`.

Reply:
See the discussion at https://github.com/Matroska-Org/matroska-specification/pull/190#discussion_r139317047

Reply:
sure, but for the moment it is not in the PR, and IMO it should be expanded with some content from the comment you point out.
(but I don't really know wich wording to use :( )

Reply:
Not sure we should repeat the name in the definition. It would be lighter to read like this:
`Matroska Player`: A data parser that interprets...

Reply:
Or simply:
A `Matroska Player` is a data parser…

Reply:
Reason for this was mimicking how it's set up in the EBML spec https://datatracker.ietf.org/doc/draft-ietf-cellar-ebml/?include_text=1